### PR TITLE
Framework and MPLS draft updates due to framework.

### PR DESCRIPTION
**Pull request by dfedyk**

### CODE REVIEW

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml [hunk 1]
```diff
@@ -258,66 +261,99 @@
...
 	 Extensibility of that Data Plane:  Whether or not the DetNet data plane
 	 includes the facility to carry additional information (metadata) that
 	 can be used to provide an enhanced service to the DetNet packet.
+          </t>
+        </list>
+         </t>
+         <t>
+          (Below replacement text suggested by Balazs)
+         </t>
+         <t>
```
Comment (louberger) [inline data-plane-framework/draft-ietf-detnet-data-plane-framework.xml:275]: this should be a comment not in the text itself

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml [hunk 2]
```diff
@@ -258,66 +261,99 @@
...
+         <list style='numbers'>
+         <t>
+          Forwarding paradigm: DetNet can be built over packet switched (IP)
+          networks or label switched (MPLS) networks .
```
Comment (louberger) [inline data-plane-framework/draft-ietf-detnet-data-plane-framework.xml:279]: how about: s/can be built over/is currently defined for operation

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml [hunk 3]
```diff
@@ -258,66 +261,99 @@
...
-              +-------+  +---------+
+  +-------+  +---------+
+  | DN IP |  | DN MPLS |
+  +-------+  +---------+
 
         ]]></artwork>
         </figure>
         <figure anchor="dn_svc_encaps" align="center"
-        title="DetNet Service Encapsulations">
+        title="DetNet Service Examples with Recursion">
```
Comment (louberger) [inline data-plane-framework/draft-ietf-detnet-data-plane-framework.xml]: can we drop "with recursion" I think this is confusing.

**Review by louberger — CHANGES_REQUESTED** [review]
please see comments above

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml (hunk 3 above)
Comment (dfedyk) [reply data-plane-framework/draft-ietf-detnet-data-plane-framework.xml]: Cheanged

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml (hunk 2 above)
Comment (dfedyk) [reply data-plane-framework/draft-ietf-detnet-data-plane-framework.xml:279]: Fixed

**Review by louberger — APPROVED** [review]

**Pull request merged**